### DISCUSSION TITLE
Not working zigbee2mqtt/bridge/config/devices/get

### DISCUSSION THREAD

**Discussion by xX-Nexus-Xx** [Koenkk/zigbee2mqtt#11294] [Devices]
Hello everyone,

I did not work on my Zigbee2Mqtt for a while and realised a few changes.  At moment I'm unable to refresh my device list, I used _**zigbee2mqtt/bridge/config/devices/get**_ for that but it is no longer working.

Can anybody tell me the MQTT cmd so I can get a refresh on Devices and Groups ?  I was not able to find in the manual

thx for your help

**Comment by lux73** [comment]
https://www.zigbee2mqtt.io/guide/usage/mqtt_topics_and_messages.html#zigbee2mqtt-bridge-devices

**Reply by xX-Nexus-Xx** [reply]
Yes, I.m aware that the devices are located in zigbee2mqtt/bridge/devices .. however this topic is only getting updated when the configuration is changed. If I miss this update or do not store info locally I'm unable to retrieve the devices list

How can I enforce that zigbee2mqtt/bridge/devices is getting re-published ??    ... earlier this was possible by zigbee2mqtt/bridge/config/devices/get

**Reply by Koenkk** [reply]
You cannot and there is no need to, `zigbee2mqtt/bridge/devices` is a retained MQTT message, your application will always receive this message upon connecting to the MQTT server.

**Reply by RobLeighton67** [reply]
Just to say - I used the same mechanism to monitor devices - in conjunction with last seen - to allow my own per-device availability checking to work.
After the application was refactored to remove this capability I modded my local copy to add the facility as part of Health Check - it works nicely.
